hack/generator/pkg/astmodel/errored_type_test.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+import (
+	"testing"
+
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/gen"
+	"github.com/leanovate/gopter/prop"
+)
+
+func TestErroredTypeProperties(t *testing.T) {
```

Comment:
Ooh nice - I need to find more ways to do property testing! I've mostly only used it in parsing-type scenarios.